content/en/docs/Testing Policies/_index.md

```diff
@@ -50,44 +50,50 @@ The following is an example of a simple GitHub Actions workflow which may be use
-3. Tests the unknown manifests in `resources/` against the known policies stored in `policies/`.
-4. Tests the pre-defined test cases stored in `tests/` which contain Kyverno test manifests, policies, and resources separated by folder.
+2. The GitHub action for Kyverno CLI is downloaded by naming a specific version. This can be changed to simulate the effect a Kyverno upgrade may have in a cluster. Please refer to [kyverno-cli-installer](https://github.com/marketplace/actions/kyverno-cli-installer) for more information on the GitHub action for Kyverno CLI.
+3. Check the Kyverno CLI version
+4. Tests the unknown manifests in `resources/` against the known policies stored in `policies/`.
+5. Tests the pre-defined test cases stored in `tests/` which contain Kyverno test manifests, policies, and resources separated by folder.
 
 ```yaml
 name: kyverno-policy-test
 on:
   - pull_request
   - workflow_dispatch
-env:
-  VERSION: v1.7.2
 jobs:
   test:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
     steps:
       - name: Checkout repo
-        uses: actions/checkout@v3
-      - name: Download Kyverno CLI
-        run: |
-          curl -sLO https://github.com/kyverno/kyverno/releases/download/${{ env.VERSION }}/kyverno-cli_${{ env.VERSION }}_linux_x86_64.tar.gz
-          tar -xf kyverno-cli_${{ env.VERSION }}_linux_x86_64.tar.gz
-          ./kyverno version
+        uses: actions/checkout@v4
+      - name: Install Kyverno CLI
+        uses: kyverno/action-install-cli@v0.2.0
+        with:
+          release: 'v1.9.5'
+      - name: Check install
+        run: kyverno version
       - name: Test new resources against existing policies
-        run: ./kyverno apply policies/ -r resources/
+        run: kyverno apply policies/ -r resources/
       - name: Test pre-defined cases
-        run: ./kyverno test tests/
+        run: kyverno test tests/
 ```
 
 Upon pull request to the repository containing this file, the GitHub Actions workflow will be triggered. If all tests succeed, the output will show a success for all steps.
 
-![ci-pass](/images/ci-pass.png)
+![ci-pass][/images/ci-pass.png]
 
 Each step may be expanded for complete output.
 
 If a pull request fails in any of the steps in the job, for example if a new manifest is introduced into `resources/` which fails one or more policies defined in `policies/`, the pipeline will halt causing the entire run to fail.
 
-![ci-fail](/images/ci-fail.png)
+![ci-fail][/images/ci-fail.png]
 
 Again, each step may be inspected for output. In the above screenshot, a Deployment manifest did not pass a validate policy and therefore caused the overall failure.
 
 For full usage details of the Kyverno CLI, refer to the [documentation](/docs/kyverno-cli/).
+
+
+[/images/ci-pass.png]: https://github.com/kyverno/kyverno/assets/103727912/54dbc4e0-ee9d-4ec5-953e-7e49ea6290af
+[/images/ci-fail.png]: https://github.com/kyverno/kyverno/assets/103727912/eba9ec26-0560-4971-92c7-ad2d30468550
```

Comment:
What is this here for? Please remove.

Reply:
I used the image URLs here. Should have just modified the images in the folder.